src/test/debuginfo/borrowed-enum.rs

```diff
@@ -18,13 +18,17 @@
 // gdb-command:run
 
 // gdb-command:print *the_a_ref
-// gdb-check:$1 = {{RUST$ENUM$DISR = TheA, x = 0, y = 8970181431921507452}, {RUST$ENUM$DISR = TheA, __0 = 0, __1 = 2088533116, __2 = 2088533116}}
+// gdbg-check:$1 = {{RUST$ENUM$DISR = TheA, x = 0, y = 8970181431921507452}, {RUST$ENUM$DISR = TheA, __0 = 0, __1 = 2088533116, __2 = 2088533116}}
+// gdbr-check:$1 = borrowed_enum::ABC::TheA{x: 0, y: 8970181431921507452}
 
 // gdb-command:print *the_b_ref
-// gdb-check:$2 = {{RUST$ENUM$DISR = TheB, x = 0, y = 1229782938247303441}, {RUST$ENUM$DISR = TheB, __0 = 0, __1 = 286331153, __2 = 286331153}}
+// gdbg-check:$2 = {{RUST$ENUM$DISR = TheB, x = 0, y = 1229782938247303441}, {RUST$ENUM$DISR = TheB, __0 = 0, __1 = 286331153, __2 = 286331153}}
+// gdbr-check:$2 = borrowed_enum::ABC::TheB(0, 286331153, 286331153)
 
 // gdb-command:print *univariant_ref
-// gdb-check:$3 = {{__0 = 4820353753753434}}
+// gdbg-check:$3 = {{__0 = 4820353753753434}}
+// FIXME
+// gdbr-check:$3 = <error reading variable>
```

Comment:
This seems fixed. Trying on trunk in a moment, but this works from a gdb I have from august.

```
(gdb) p univariant_ref
$1 = (borrowed_enum::Univariant *) 0x7fff5fbff820
(gdb) ptype univariant_ref
type = union borrowed_enum::Univariant {
    borrowed_enum::TheOnlyCase;
} *
```